### PR TITLE
47-feature/dashboard navigation

### LINKED ISSUE #47: Component: Dashboard navigation

### PR DESCRIPTION
**Pull request by joakimunge**
Adds Sidebar navigation component. For authenticated users on dashboard.

Closes #47

### CODE REVIEW

File: src/Components/Navigation.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+const Navigation = ({ styles, ...props }) => (
+  <aside {...css(styles.sidebar)} {...props}>
+    <div {...css(styles.logoArea)}>
+      <Heading {...css(styles.logo)} size="2">
+        Feed<span>.io</span>
+      </Heading>
+    </div>
+    <span {...css(styles.divider)} />
+    <FlexContainer align="start">
+      <ul {...css(styles.dashboardNav)}>
```
Comment (Ciavarella) [inline src/Components/Navigation.js]: Use ListItem and List component instead? Otherwise it looks good!

Comment (joakimunge) [reply src/Components/Navigation.js]: I opted against using those due to this being a very specific edge case. Using general components in this instance would introduce more headaches, rather than not.

**Comment by Andreas-sjolund-chas** [comment]
Test
